justfile

```diff
@@ -1,12 +1,9 @@
 set dotenv-load := true
 
-# just has no idiom for setting a default value for an environment variable
-# so we shell out, as we need VIRTUAL_ENV in the justfile environment
-export VIRTUAL_ENV  := `echo ${VIRTUAL_ENV:-.venv}`
+export VIRTUAL_ENV  := env_var_or_default("VIRTUAL_ENV", ".venv")
```

Comment:
`env_var_or_default`

I didn't know that Just provided this. It looks very useful. Thank you!